docs/content/en/blog/articles/dev-config/index.md

```diff
@@ -0,0 +1,321 @@
+
+The takeaway here: **leaving heap settings blank is not a good idea for a dev environment** in particular, where several Cassandra instances will be collocated on the same host machine. (By default, K8ssandra does not allow multiple Cassandra nodes on the same Kubernetes worker node. For this post, we’re using kind to have multiple worker nodes run on the same OS instance - or virtual machine in the case of Docker Desktop).
+
+The chosen heap size will directly impact the throughput you can expect to achieve (although it’s not the only limiting factor). A small heap will involve more garbage collections, which will generate more stop the world pauses and directly impact throughput and latency. It also increases the odds of running out of memory if the workload is too heavy, as objects cannot end their lifecycle fast enough for the available heap space.
+
+Setting the heap size at 500MB with 200MB of new gen globally for the cluster would be done as follows:
+
+
+```
+cassandra:
+  version: "3.11.10"
+  ... 
+  ...
+  # Cluster level heap settings
+  heap: 
+   size: 500M
+   newGenSize: 200M
+
+  datacenters:
+  - name: dc1
+    size: 3
+```
+
+### Stargate 
+Because Stargate nodes are special coordinator-only Cassandra nodes and run in the JVM, it is also necessary to set their max heap size:
+
+```
+stargate:
+  enabled: true
+  version: "1.0.9"
+  replicas: 1
+  ...
+  ...
+  heapMB: 256
+```
+
+Stargate nodes will follow the same rule when it comes to off heap memory: the JVM will be allowed to use as much RAM for off heap memory as the configured heap size. 
+
+As Stargate serves as coordinator, it is likely to hold objects for longer on heap waiting for all nodes to respond to queries before it can acknowledge them and potentially return the result sets to clients. It needs enough heap to do so without excessive garbage collection.
+Unlike Cassandra, Stargate doesn’t compute a heap size based on the available RAM, and the value must be set explicitly.
+
+During our tests, we observed that 256MB was a good initial value to have stable Stargate pods. In production you might want to tune this value for optimal performance.
+
+# Benchmark Environment
+Our setup for running benchmarks was the following:
+* Apple MacBook Pro 2019 - i7 (6 cores) - 32GB RAM - 512GB SSD
+* Docker desktop 3.1.0
+* [Kind](https://kind.sigs.k8s.io/) 0.7.0
+* Kubernetes 1.17.11
+* kubectl v1.20.2
+
+Note that we’ve used a fairly powerful environment as our tests ran on a 2019 Apple MacBook Pro with a 6 cores i7 CPU and 32GB RAM.
+
+We used the [Kind deployment guidelines]({{< relref "docs/topics/ingress/traefik/kind-deployment/">}}) found in the [K8ssandra documentation]({{<relref "docs">}}) to start a k8s cluster with 3 worker nodes.
+
+Docker Desktop allows to tune its allocated resources by clicking on its icon in the status bar, then going to “Preferences…”:
+
+<img alt="Docker Preferences" src="docker-preferences.png" width="250" />
```

Comment:
Are you using the `<img>` tag here instead of `![]()` to set the width to `250`?

Reply:
yes, otherwise looks too large